File: app/src/main/java/hottopic/mit/co/nz/cleaningservice/entities/network/response/BaseResponse.java
package hottopic.mit.co.nz.cleaningservice.entities.network.response;

import java.io.Serializable;

/**
 * The base response entity
 */
public class BaseResponse implements Serializable{
    private int code;//The response code from backend server
    private String message;//The response message from backend server

    public int getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }
}

File: app/src/main/java/hottopic/mit/co/nz/cleaningservice/entities/network/response/BaseResponse.kt
package hottopic.mit.co.nz.cleaningservice.entities.network.response

import java.io.Serializable

/**
 * The base response entity
 */
open class BaseResponse : Serializable {
    val code: Int = 0//The response code from backend server
    val message: String = ""//The response message from backend server
}
